pkg/webui/console/store/reducers/pagination.js

```diff
@@ -33,6 +36,12 @@ export const createNamedPaginationReducer = function (reducerName, entityIdSelec
         totalCount: payload.totalCount,
         ids: payload.entities.map(entityIdSelector),
       }
+    case DELETE_PAGINATION_SUCCESS:
+      return {
+        ...state,
+        totalCount: state.totalCount - 1,
```

Comment:
```suggestion
        totalCount: Math.max(0, state.totalCount - 1),
```

Just an idea. Might catch some issues in rare cases, but I don't know if this is the place to do this.